content/issues/2021-09-08-outage.md

```diff
@@ -1,9 +1,10 @@
 ---
 title: "September 8th outage"
 date: 2021-09-07T20:30:00+02:00
-resolved: false
+resolved: true
```

Comment:
I would suggest not to mark this as resolved, as we are still working on adding one more worker to make SRPM builds faster. Until this is done, users might still periodically experience slow SRPM builds, and they would complain about this on IRC (as they did last week). Changing the title to "Slow SRPM builds" could probably ease some of the confusion and better reflect the actual situation.